docs/shared/upp.yaml

```diff
@@ -0,0 +1,41 @@
+upp:
+  execution:
+    batchargs:
+      nodes: 1
+      export: NONE
+      walltime: 00:05:00
+    envcmds:
+      - module use /path/to/modulefiles
+      - module load runtime-module
+      - source /etc/profile.d/slurm.sh
+    executable: /path/to/upp.x
+    mpiargs:
+      - "--ntasks $SLURM_CPUS_ON_NODE"
+    mpicmd: srun
+  files_to_copy:
+    postxconfig-NT.txt: /path/to/postxconfig-NT.txt
+  files_to_link:
+    eta_micro_lookup.dat: /path/to/nam_micro_lookup.dat
+    params_grib2_tbl_new: /path/to/params_grib2_tbl_new
```

Comment:
Hardlinks are a great idea, but I think they need to be opt-in, at least as a replacement for symlinks, because a user expecting symlinks will probably be expecting the provenance information implied e.g. by the RHS of the `->` info info shown by `ls -l` to be preserved. It might be appropriate to silently hardlink instead of copying when possible, though. I don't immediately see a downside to that.